json/json.go

```diff
@@ -68,7 +68,7 @@ func (s *jsonSearcher) jsonSearch(w http.ResponseWriter, req *http.Request) {
 		searchArgs.Opts = &zoekt.SearchOptions{}
 	}
 
-	q, err := query.Parse(searchArgs.Q)
+	q, err := query.Parse(searchArgs.Q, false)
```

Comment:
by default we want to disable regex for everything existing in zoekt?

Reply:
we can run a webserver via zoekt. We are not using this feature

Reply:
On a second thought, keeping not usable things to their previous state which is regex enabled by default.